media-sound/amc/amc-1.14.0.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A'dam Music Composer"
+DESCRIPTION="This Mini-Scheme Interpreter is based on SCHEME Interpreter in \
+Common Lisp in Appendix of T.Matsuda & K.Saigo, Programming of LISP, \
+archive No5 (1987) p6 - p42 (published in Japan)."
+HOMEPAGE="https://github.com/HaikuArchives/AMC"
+COPYRIGHT="1989 Atsushi Moriwaki "
+LICENSE="Public Domain"
+REVISION="1"
+srcGitRev="451087418bc91f626ba7cb1b112816e3bac898f9"
+SOURCE_URI="https://github.com/HaikuArchives/AMC/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="8e90759b128c66f3a3443fecf0b9ffda112a911be77fc14d7666d7bb051b98f7"
+SOURCE_DIR="AMC-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2"
+
+PROVIDES="
+	amc = $portVersion
+	app:AMC = $portVersion
+	"
+
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:make
+	cmd:gcc
+	"
+
+BUILD()
+{
+	cd amc/src
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd amc/src
+	mkdir -p $appsDir/AMC
+	cp -a amc $appsDir/AMC/A\'dams\ Music\ Composer
```

Comment:
There was a discussion a long time ago about what should go to the $appsDir.
My opinion was/is that we should not put subdirectories in $appsDir, but only runtimes.
The rarionale for this is that $appsDir is in $PATH, so putting any app in a subdirectory of $appsDir means that you cannot launch that app from the command line (unless you give its full path.)
The problem with my approach is that some apps require that their data reside on the same directory as the runtime itself, but enforcing this would lead to a crowded $appsDir.
The ideal solution would be to put the data in $dataDir/AMC/ and the runtime in $appsDir, using AMC for the name of the runtime. But I don't know if this works for AMC. (I did not try to build it yet.)
I had given up trying to convince the other contributors about this, and I don't plan to try it again.

I just started writing this because I think the runtime should be called AMC, not "A'dams Music Composer", and here am I, bringing back an old topic I don't really care about.
So please feel free to ignore this message.